app/models/periodictask.rb

```diff
@@ -1,6 +1,7 @@
 class Periodictask < ActiveRecord::Base
   unloadable
   belongs_to :project
+  belongs_to :assigned_to, :class_name => 'Principal', :foreign_key => 'assigned_to_id'
```

Comment:
IMHO not, this does not change anything in database. It just links to class Principal (users-table) via foreign-key. As i did not found any database-migration examples for foreign-keys in redmine source i think, this does not need any migration. 

I did not migrate. Just clone master, apply patch and restart redmine.